presto-main/src/main/java/com/facebook/presto/memory/QueryContext.java

```diff
@@ -68,6 +68,8 @@
     private final long maxSpill;
     private final SpillSpaceTracker spillSpaceTracker;
     private final Map<TaskId, TaskContext> taskContexts = new ConcurrentHashMap();
+    // Corresponding to AbstractAggregatedMemoryContext.FORCE_FREE_TAG
```

Comment:
Please directly use `AbstractAggregatedMemoryContext.FORCE_FREE_TAG`. It's more error proof that way.

Reply:
> Please directly use `AbstractAggregatedMemoryContext.FORCE_FREE_TAG`. It's more error proof that way.

They're in different project, so if we want to directly use, we need to add maven dependency. Should we add maven dependency or just use this this way?

Reply:
AbstractAggregatedMemoryContext.FORCE_FREE_TAG is in presto-memory-context